### PR TITLE
Config port change

### PR DESCRIPTION
**Pull request by subashsn**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -27,10 +32,11 @@ class MQTTFacility extends BaseFacility {
 
     const srv = require('net').createServer(aedes.handle)
     this.server = srv
+    this.aedes = aedes
```
Comment (vigan-abd) [inline index.js:35]: what's the purpose of binding this one to class instance?

Comment (subashsn) [reply index.js:35]: We are accessing this from the container worker
https://github.com/PearTree-to/moria-wrk-container-bitdeer/blob/28390e8714a318a5bc67c7964eff9bf86299650f/workers/lib/worker-base.js#L66

Comment (vigan-abd) [reply index.js:35]: don't have access here

Comment (vigan-abd) [reply index.js:35]: in that case why not also use same on  const srv = require('net').createServer(aedes.handle)

Comment (chetasr) [reply index.js:35]: I can, didn't use it because it's the same instance anyways. Will update if required.

Comment (subashsn) [reply index.js:35]: @vigan-abd Is this resolved?

Comment (vigan-abd) [reply index.js:35]: @chetasr  @subashsn not a blocked but I think would be cleaner to use this.aedes everywhere

**Pull request merged**